File: Data Structures & Algorithms/missing-number/submission-1.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        if nums and len(nums) > 0  :
            if nums[0] != 0:
                return 0
            for i,v in enumerate(nums):
                if i < len(nums) -1:
                    if nums[i+1] - v == 2 :

                        return v+1
        return len(nums)
        

File: Data Structures & Algorithms/is-anagram/submission-1.cs
public class Solution {
    public bool IsAnagram(string s, string t) {

        if (s.Length != t.Length )
        {
            return false;
        }
        char[] s1 = s.ToCharArray();
        Array.Sort(s1);

        char[] t1 = t.ToCharArray();
        Array.Sort(t1);



        for (int i = 0; i< s.Length ; i++)
        {
            if (s1[i] != t1[i])
            {
                return false;
            }
        }


        return true;
    }
}
